### DISCUSSION TITLE
Binding - Abhängig von DP

### DISCUSSION THREAD

**Discussion by AFPass** [Zefau/ioBroker.jarvis#1384] [Q&A]
Ich habe ein Gerät mit mehreren DP.
Gerät.DP1
Gerät.DP2
Gerät.DP3

Ich möchte bei einem DP die Anzeige ändern in Abhängigkeit von einem DP zu diesem Gerät.

Ich weiß das ich mit Bindings den DP-Namen aus ioBroker nutzen kann.
Aber kann ich auch direkt auf Gerät.DP2 "binden"?

Also in DP1:
{"{DP2}=0":{"visibility":"hidden"}}
statt:
{"{0_userdata.0.testwert}=0":{"visibility":"hidden"}}

**Comment by Zefau** [comment]
Hilft dir https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/binding#binding ?

**Reply by mcuiobroker** [reply]
Die Frage ist, ob man es als Feature Request ansieht, damit man auch die Werte von den Geräte-DP innerhalb eines Gerätes direkt als Binding mit nutzen kann.
Derzeit ist es nicht möglich.

**Reply by AFPass** [reply]
> Hilft dir https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/binding#binding ?

Nein. Das kenne ich. Wie @mcuiobroker im nächsten Post sagt, bezieht sich der DP auf einen DP innerhalb des Gerätes.

Also dann eher ein FeatureRequest?

**Reply by Zefau** [reply]
Dem Binding ist es egal ob gleiches Gerät oder nicht.

> Ich weiß das ich mit Bindings den DP-Namen aus ioBroker nutzen kann.

Jo. Warum reicht das nicht?

> Aber kann ich auch direkt auf Gerät.DP2 "binden"?

Geht nicht, nein.
